composite/token-allocation/src/adapter.ts

```diff
@@ -132,7 +110,7 @@ export const execute = async (input: AdapterRequest, config: Config): Promise<Ad
       // eslint-disable-next-line no-case-declarations
       const price = await computePrice(config, allocations, quote)
       return _success(price.payload, price.result)
-    case 'marketcap':
+    case 'marketCap':
```

Comment:
@boxhock didn't you lowercased this recently in a hotfix IIRC?

Reply:
Yeah. We should convert the input to lowercase and then compare that